test-suite/src/run-test-suite.ts

```diff
@@ -202,6 +202,17 @@ export const runTestSuite = (params: TestSuiteParams) => {
                     assert.deepEqual(await driver.$('.todo-item').attr('data-value'), '');
                 });
             });
+
+            it('should get correct attribute [when] $ is deep nested', async () => {
```

Comment:
```suggestion
            it('gets correct attribute [when] $ is deep nested', async () => {
```
(http://bretthard.in/post/tests-should-not-begin-with-should)